unpackaged/config/trial_tso/prompts/NPSPContactMergeGuidance.prompt

```diff
@@ -3,7 +3,7 @@
     <masterLabel>NPSP Contact Merge Guidance</masterLabel>
     <promptVersions>
         <actionButtonLabel>Find and Merge Duplicates</actionButtonLabel>
-        <actionButtonLink>https://powerofus.force.com/s/article/NPSP-Merging-Contacts</actionButtonLink>
+        <actionButtonLink>https://help.salesforce.com/s/articleView?id=sf.contacts_merge.htm&amp;type=5</actionButtonLink>
```

Comment:
I replaced this based on assumptions, I think a large part of this button is based on the link so I wasn't super sure if we wanted to remove it entirely, but can remove if seen as appropriate!

Reply:
This link is more specific to NPSP: https://help.salesforce.com/s/articleView?id=sfdo.NPSP_Merge_Contacts.htm&type=5

Reply:
Thank you for pointing that out, have made an update 👍